src/index.js

```diff
@@ -96,24 +98,131 @@ export async function getArgs () {
       console.log(result)
     })
 
+  program
+    .command('installAppBundle <InstalledAppId> <AgentHash> <AppBundleSource>')
+    .alias('b')
+    .option('-e --membraneProof <membraneProof>', 'provide membrane proof for app bundle install and runtime validation - should be paired with --cellNick')
+    .option('-n --cellNick <cellNick>', 'provide cell nick - should be paired with --membraneProof')
+    .option('-u --uid <uid>', 'provide uid to happ')
+    .description('install provided happ bundle with given id and details: calls InstallAppBundle(installed_app_id, agent_key, source, membrane_proofs?, uid?) -> { installed_app_id, cell_data: [ { cell_id, cell_nick } ], status: { inactive: { reason: [Object] } } }')
+    .action(async (InstalledAppId, AgentHash, AppBundleSource, { membraneProof, cellNick, uid }) => {
+      // throw error if one, but now both, of the membraneProof and CellNick are present - both are needed to form the membrane_proofs object
+      if (!membraneProof ^ !cellNick) {
+        throw new Error('When installing a happ with membrane proof, both the --membraneProof and --cellNick options are required.')
+      }
+
+      let bundleSource
+      // eslint-disable-next-line no-useless-escape
+      const urlMatcher = /https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)?/gi
+      if (AppBundleSource.match(urlMatcher)) {
+        if (path.extname(AppBundleSource) !== '.happ') {
+          throw new Error('Invalid AppBundleSource URL.  Please ensure to provide the url address of your bundle\'s .happ file.')
+        }
+        bundleSource = AppBundleSource
+      } else {
+        try {
+          const isHappExt = path.extname(AppBundleSource) === '.happ'
+          if (!isHappExt) throw new Error('Invalid AppBundleSource file path.  Please ensure to provide the url address of your bundle\'s .happ file.')
+        } catch (error) {
+          throw new Error('Invalid AppBundleSource. The source must be either a file or url path to the .happ file.')
+        }
+        const bundlePath = path.join(__dirname, AppBundleSource)
+        bundleSource = bundlePath
+      }
+
+      const membraneProofs = {
+        proof: Buffer.from(membraneProof, 'base64'),
+        cell_nick: cellNick // 'elemental-chat'
```

Comment:
Latest conductor-api code looks like the format for this should be 
```suggestion
      const membraneProofs = {
        [[cell_nick]]: Buffer.from(membraneProof, 'base64'),      
```

see this line: https://github.com/holochain/holochain-conductor-api/blob/8967b93d3e3eace63110b20ea2765112e3388d54/src/api/admin.ts#L133

I haven't investigated if this is a new change in conductor-api, if it is please ignore this suggestion :)

Reply:
Good catch. :medal_sports: Updated.